content/04.results.md

```diff
@@ -1,6 +1,23 @@
 ## Results
 
-### Comparing bioRxiv to PubMed Central
+### Comparing bioRxiv to PubMed Central
+
+#### bioRxiv Repository
+
+![
+BioRxiv has an influx of neuroscience preprints followed by bioinformatics and then microbiology.
+This bar chart depicts the number of preprints that fall into each category of bioRxiv.
+](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/exploratory_data_analysis/output/figures/preprint_category.png){#fig:biorxiv_categories}
+
+![
+Majority of bioRxiv's articles report new research findingds.
+This bar char depicts the number of articles categorized based on report findings.
+](https://raw.githubusercontent.com/greenelab/annorxiver/94874e0f1e35bd667bf3b7c3dc6416068779444f/biorxiv/exploratory_data_analysis/output/figures/preprint_headings.png){#fig:biorxiv_headings}
+
+BioRxiv allows users to specify the most appropiate topic for their preprint submission along with the type of finding being reported.
+Majority of research findings report new results (Figure {@fig:biorxiv_headings}).
+Regarding topic categories, majority of preprints fall within the neuroscience field (Figure {@fig:biorxiv_categories}).
```

Comment:
How does this compare with previous descriptions of the composition of bioRxiv, potentially done at earlier times?